cmd/bucket-handlers.go

```diff
@@ -448,11 +448,22 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 	w.Header().Set("ETag", "\""+objInfo.MD5Sum+"\"")
 	w.Header().Set("Location", getObjectLocation(bucket, object))
 
-	// Set common headers.
-	setCommonHeaders(w)
+	// Decide what http response to send depending on success_action_status parameter
+	switch formValues["Success_action_status"] {
```

Comment:
Perhaps you can do http.Canonical so that it gets easier to understand the purpose.